typespec-extension/src/utils.ts

```diff
@@ -147,21 +146,11 @@ export async function loadExamples(program: Program, options: EmitterOptions): P
   return operationExamplesMap;
 }
 
-export function resolveOperationId(program: Program, operation: Operation) {
-  // const explicitOperationId = getOperationId(program, operation);
-  // if (explicitOperationId) {
-  //   return explicitOperationId;
-  // }
```

Comment:
this would be handled by openapi lib, for `@operationId` though it is not recommended.